kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/symbol/kotlin/KSTypeReferenceImpl.kt

```diff
@@ -71,7 +72,11 @@ class KSTypeReferenceImpl(private val ktType: KtType) : KSTypeReference {
     }
 
     override val modifiers: Set<Modifier>
-        get() = TODO("Not yet implemented")
+        get() = if (ktType is KtFunctionalType && ktType.isSuspend) {
+            setOf(Modifier.SUSPEND)
```

Comment:
Can we leave a todo in the comment to address other possibilities? Looks like it only handled a special case for the test.

Reply:
We only treat `SUSPEND` modifier for type reference in FE1.0 implementation.